### PR TITLE
Make missing strings translatable, update locale files and German

### PR DESCRIPTION
**Pull request by Wuzzy2**
This PR does the following:

- Mark a bunch of strings in this mod as translatable by adding the `S()`
- Update locale files
- Update German translation

### CODE REVIEW

**Review by BuckarooBanzay — APPROVED** [review]

File: api.lua [hunk 1]
```diff
@@ -1343,14 +1344,14 @@ function get_sign_formspec(pos, nodename)
...
-		"label[0.3,3.4;Unicode font]",
+		"label[0.3,3.4;"..FS("Unicode font").."]",
 		"image_button[0.6,3.7;1,0.6;signs_lib_switch_" .. state .. ".png;uni_"
 			.. state .. ";;;false;signs_lib_switch_interm.png]",
 	}
 
 	if minetest.registered_nodes[nodename].allow_widefont then
 		state = meta:get_int("widefont") == 1 and "on" or "off"
-		formspec[#formspec+1] = "label[2.1,3.4;Wide font]"
+		formspec[#formspec+1] = "label[2.1,3.4;"..FS("Wide font").."]"
```
Comment (OgelGames) [inline api.lua:1354]: Why do these need to be escaped? I thought escaping was only needed for user-editable values.

File: api.lua [hunk 2]
```diff
@@ -1343,14 +1344,14 @@ function get_sign_formspec(pos, nodename)
 		"image[0.1,2.4;7,1;signs_lib_sign_color_palette.png]",
 		"textarea[0.15,-0.2;6.3,2.8;text;;" .. minetest.formspec_escape(txt) .. "]",
 		"button_exit[3.7,3.4;2,1;ok;" .. S("Write") .. "]",
-		"label[0.3,3.4;Unicode font]",
+		"label[0.3,3.4;"..FS("Unicode font").."]",
```
Comment (Niklp09) [inline api.lua:1347]: same as https://github.com/mt-mods/signs_lib/pull/19/files#r1251892735

File: api.lua (hunk 1 above)
Comment (SwissalpS) [reply api.lua:1354]: translated strings are almost user-editable. But you may be right that translator already escapes them?

**Comment by Wuzzy2** [comment]
They must be escaped because the translation might include symbols that must be escaped.
doing formspec escapes for translations is standard practice in minetest.
We should not expect the translator to do the formspec escapes for us.

**Comment by OgelGames** [comment]
I'd say those should be escaped in the translation if it were ever necessary to include them, but I guess either way works ¯\\\_(ツ)\_/¯

**Comment by SwissalpS** [comment]
> I'd say those should be escaped in the translation if it were ever necessary to include them, but I guess either way works ¯\_(ツ)_/¯

We can't expect translators to be aware of such quirks. Better to just let them type and handle the technical stuff on the code-end.

**Pull request merged**